text/0560-integer-overflow.md

```diff
@@ -125,10 +125,14 @@ The error conditions that can arise, and their defined results, are as
 follows. The intention is that the defined results are the same as the
 defined results today. The only change is that now a panic may result.
 
-- The operations `+`, `-`, `*`, `/`, `%` can underflow and
-  overflow.
+- The operations `+`, `-`, `*`, can underflow and overflow. When checking is
+  enabled this will panic. When checking is disabled this will two's complement
+  wrap.
+- The operations `/`, `%` are nonsensical for the arguments `INT_MIN` and `-1`.
+  When this occurs there is an unconditional panic.
```

Comment:
Seems to [work for me](https://play.rust-lang.org/?gist=f9cb12e7c3b1c3ac5b20&version=nightly).